test/plug/adapters/cowboy/conn_test.exs

```diff
@@ -113,13 +113,11 @@ defmodule Plug.Adapters.Cowboy.ConnTest do
   end
 
   def send_418(conn) do
-    conn
-    |> send_resp(418, "")
+    send_resp(conn, 418, "")
   end
 
   def send_451(conn) do
-    conn
-    |> send_resp(451, "")
+    send_resp(conn, 451, "")
```

Comment:
This is wildly debatable but I thought we could drop the pipe since it's a single function call, but I can revert if others don't like this change :)